systemtest/src/test/java/io/strimzi/systemtest/RollingUpdateST.java

```diff
@@ -777,6 +778,41 @@ void testTriggerRollingUpdateAfterOverrideBootstrap() throws CertificateExceptio
         // TODO: send and recv messages via this new bootstrap (after client builder) https://github.com/strimzi/strimzi-kafka-operator/pull/2520
     }
 
+    @Test
+    void testAccidentallyRemovedCaTriggersRollingUpdate() {
```

Comment:
Might be overly nit-picky, but the cause of why it is removed could be seen as not part of the 'test'
I would opt to have it be called something like
```
testClusterCaRemovedTriggersRollingUpdate
```
and then have a documented comment explaining the context of this test is it being accidentally removed.